system/group.py

```diff
@@ -410,8 +425,10 @@ def main():
                 module.fail_json(name=group.name, msg=err)
 
     elif group.state == 'present':
+        if group.force == True:
+            (rc, out, err) = group.group_mod(gid=group.gid, newname=group.name, force=group.force)
```

Comment:
@b3rn, support for *check_mode* should be added to avoid modification when it's active.

Also, `newname` parameter is ignored in `Group#group_mod` implementation.

Furthermore, I'd suggest to rename `force` parameter to something like `rename` since it seems to be not idempotent operation or better rewrite it to be one.